mardi_importer/mardi_importer/zbmath/ZBMathSource.py

```
@@ -520,49 +520,42 @@ def push(self):
-                    if (
-                        self.conflict_string in info_dict["author"]
-                        or "None" in info_dict["author"]
-                    ):
-                        author_strings = [None] * len(author_ids)
+                author_ids = [None if x.strip() == "None" or self.conflict_string in x else x.strip()
+                                for x in info_dict["author_ids"].split(";")]
+                author_strings = [None if x.strip() == "None" or self.conflict_string in x else x.strip()
+                                for x in info_dict["author"].split(";")]
+                authors = []
+                author_name_strings = []
+                for a, a_id in zip(author_strings, author_ids):
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)